modules/tracking/perf/perf_Tracker.cpp

```diff
@@ -343,3 +343,73 @@ PERF_TEST_P(tracking, tld, testing::Combine(TESTSET_NAMES, SEGMENTS))
   SANITY_CHECK( bbs_mat, 15, ERROR_RELATIVE );
 
 }
+
+PERF_TEST_P(tracking, goturn, testing::Combine(TESTSET_NAMES, SEGMENTS))
```

Comment:
It is better to use `GOTURN` identifier here to be consistent with accuracy test (or use `goturn` in accuracy tests).